Makefile

```diff
@@ -1,70 +0,0 @@
-include include.mk
```

Comment:
Could we leave the `Makefile`? Some of the targets are useful for developers, such as `stylish` and `docs`. We could remove all the Jenkins-specific stuff, though.

Reply:
I'm slightly opposed to using a `Makefile` when the targets aren't directly files, because then you end up over-building or under-building some of the phony targets. I can put it back, but was wondering if you'd be ok with changing `./script/jenkins => ./scripts/build` and making it have subcommands like `./scripts/build [clean|check|stylish|build|test|docs]`, then the Jenkins script will just do:

```
#!/usr/bin/env bash
set -e

./scripts/build clean
./scripts/build check
./scripts/build stylish
./scripts/build build
./scripts/build test
./scripts/build docs
```

Can also put it at `./build` instead of `./scripts/build`.

Let me know which you prefer, I'll get it in ASAP.

Reply:
One argument against multiple scripts is that we'll have to make sure any environment setup they have is synchronized. We could do this by having a `./build/env` which is `source`d by the other scripts.

If we do it as one script, I would use this style https://github.com/kframework/evm-semantics/blob/82c71231f74ed96d042494b680105e1a996235ed/kevm#L199. We lose tab-completion, but it is fairly straightforward/lightweight.

Reply:
> We could do this by having a `./build/env` which is `source`d by the other scripts.

I think this is fine. It also has the advantage that a developer could source that file into their own shell to do debugging.

Reply:
Going to move this discussion to a future PR.